.github/workflows/build.yml

```diff
@@ -62,6 +63,13 @@ jobs:
             echo "CC=clang-${{ matrix.version }}" >> $GITHUB_ENV
             echo "CXX=clang++-${{ matrix.version }}" >> $GITHUB_ENV
           fi
+      - name: Install and Test (Linux)
+        if: runner.os == 'Linux'
+        # Sudo is required for Ubuntu 18.04 on GitHub.
+        # Otherwise "mkdir -p build/Testing/Temporary" fails with
+        # "mkdir: cannot create directory ‘build/Testing’: Permission denied"
+        # (mkdir itself was a test after "ctest" failed to create the directory)
+        run: (sudo ./install.sh && sudo make tests test-installation)
```

Comment:
Can we get rid of sudo? I think we don't have Ubuntu 18.04 anymore. If we can, it should be possible to deduplicate here.